hummingbot/connector/exchange/chainflip_lp/chainflip_lp_constants.py

```diff
@@ -0,0 +1,72 @@
+from hummingbot.core.api_throttler.data_types import RateLimit
+
+DEFAULT_DOMAIN = "chainflip_lp"
+EXCHANGE_NAME = "Chainflip"
+
+REST_RPC_URLS = {
+    "chainflip_lp": "https://mainnet-rpc.chainflip.io",
+    "chainflip_lp_testnet": "https://archive.perseverance.chainflip.io",
+}
+WS_RPC_URLS = {
+    "chainflip_lp": "wss://mainnet-rpc.chainflip.io",
+    "chainflip_lp_testnet": "wss://archive.perseverance.chainflip.io",
+}
+WS_HEARTBEAT_TIME_INTERVAL = 30
+
+LISTENER_TIME_INTERVAL = 10
+
+MAX_REQUEST = 1000
+SECOND = 1
+MAX_ID_LEN = (2**64) - 1
+
+
+# Public chainflip lp rpc methods
+ACTIVE_POOLS_METHOD = "cf_pool_environment"
+ASSET_BALANCE_METHOD = "lp_asset_balance"
```

Comment:
```suggestion
ASSET_BALANCE_METHOD = "lp_asset_balances"
```

The correct method is https://docs.chainflip.io/lp/integrations/lp-api#lp_asset_balances